app/views/past_foreign_secretaries/austen_chamberlain.html.erb

```diff
@@ -1,52 +1,49 @@
-      <p>Although Locarno was considered a success at the time, later criticism pointed out that it did not deliver peace. Poland and Czechoslovakia were concerned that the lack of a mutual assurance treaty for their borders amounted to an invitation to invade. Chamberlain had not supported the inclusion of Germany&rsquo;s eastern borders; it was, he said (adapting Bismarck&rsquo;s famous phrase), something for which &ldquo;no British government ever will or ever can risk the bones of a British Grenadier&rdquo;.</p>
-      <h3>The final years in office</h3>
-      <p>Chamberlain was in the Foreign Office for nearly 4 years after Locarno was signed, but had few other notable achievements. His time was taken up with issues outside Europe, in particular China and Egypt. It is not clear whether his interest in politics was ever so strongly engaged again.</p>
-      <p>During the post&ndash;Locarno years his health deteriorated. Diplomatic relations with the Americans, Egyptians, Chinese and Soviets degenerated, although this cannot be wholly attributed to Chamberlain. He left office in 1929 with the change of government, but always wanted to return to the Foreign Office.</p>
-      <p>Austen Chamberlain had a number of strengths, but some of these had unfortunate consequences. He behaved with integrity, but was taken by surprise when others failed to do so. He was intrinsically loyal, but would often back particularly unpopular people, or give too much leeway to people he liked, such as French Foreign Minister Briand. He fostered good relations with the French, who trusted him, but was dismissive of the Americans. His attention to detail included a desire for control that meant that he undermined others, such as his colleague Robert Cecil. In effect, he was very supportive and helpful to friends, but effectively blind to everything and everyone else.</p>    
+      <p class="govuk-body govuk-!-font-weight-bold">&ldquo;In appearance, costume, method of speech, he seemed almost a survival. His top hat, his eyeglass, his exquisite courtesy and his rotund oratory marked him out from his colleagues.&rdquo;</p>
+
+      <p class="govuk-body">These were the words of one backbencher about Austen Chamberlain, a Conservative statesman and former Foreign Secretary, in his later years. His career was overshadowed firstly by the character of his father, Joseph Chamberlain, a pioneering Lord Mayor of Birmingham and prominent tariff reformer, and later by the &lsquo;peace in our time&rsquo; that his half-brother Neville, Prime Minister between 1937 and 1940, tried to ensure.</p>
+
+      <h3 class="govuk-heading-s govuk-!-margin-0">The Locarno Pact</h3>
```

Comment:
This heading looks a little too close to the text beneath it, but I assume this will be looked at when it's swapped for a component?

Reply:
That's the existing spacing, so changing it is out of scope.

For example:
<img width="291" alt="Screenshot 2019-10-25 at 11 08 27" src="https://user-images.githubusercontent.com/1732331/67563027-cd2b8280-f717-11e9-80a3-efea91a0c5ff.png">

Yep, all of this is set to be component-ised at a later date.